subcommands/devices/config_updates.go

```diff
@@ -94,6 +128,82 @@ func validateUpdateArgs() {
 		fmt.Println("       apps must be ", pattern)
 		os.Exit(1)
 	}
+
+	tags := device.Tags
+	if len(updateTags) > 0 {
+		tags = strings.Split(updateTags, ",")
+	}
+	apps := device.DockerApps
```

Comment:
Will it work for Compose Apps?

Reply:
yes. "docker apps" as a device attribute means what array of apps its reporting up. so its package manager agnostic.